WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m

```diff
@@ -177,11 +166,20 @@ - (void)configureFeaturedImage
     self.labelsContainerTrailing.active = !hideFeaturedImage;
     
     if (!hideFeaturedImage) {
-        [self.featuredImageLoader loadImageFromMedia:page.featuredImage
-                                       preferredSize:CGSizeMake(FeaturedImageSize, FeaturedImageSize)
-                                         placeholder:nil
-                                             success:nil
-                                               error:nil];
+        [self.featuredImageView startLoadingAnimation];
+
+        __weak __typeof(self) weakSelf = self;
+
+        [page.featuredImage imageWithSize:CGSizeMake(FeaturedImageSize, FeaturedImageSize)
+                        completionHandler:^(UIImage * _Nullable result, NSError * _Nullable error) {
+                            __strong __typeof(weakSelf) strongSelf = weakSelf;
+                            [strongSelf.featuredImageView stopLoadingAnimation];
+                            
+                            if (error == nil) {
```

Comment:
I might have missed it, but I didn't see where we were logging the error anywhere.  Maybe we could add call to DDLogError when there is an error to help with any future debugging should there be a need?